listenbrainz/webserver/templates/playlists/playlists.html

```diff
@@ -1,4 +1,12 @@
 {% extends 'user/base.html' %}
+{%- block title -%}
+  {% if current_user.is_authenticated %}
+  Your 
+  {% else %}
+  {{ user.musicbrainz_id }}'s
+  {%- endif -%}
```

Comment:
This isn't exactly what we need.
`current_user.is_authenticated` will be true if a user is logged in. What we want this condition to check is if `current_user.musicbrainz_id == user.musicbrainz_id`

Of course this will apply to other pages as well.

Reply:
Got it. I will implement the suggested. Thank you for the help.